PREVIEW-TEMPLATING.md

```diff
@@ -465,6 +465,7 @@ This is the full list of document properties that were added to allow the desire
 | `vulnerabilities[].product_status.known_not_affected[].name`          | The value should be the product’s full canonical name, including version number and other attributes, as it would be used in a human-friendly document.                                                                                                         |
 | `vulnerabilities[].product_status.known_not_affected[].vectorString`  | The vectorString of the score in which this product_status is referenced by id.                                                                                                                                                                                 |
 | `vulnerabilities[].product_status.known_not_affected[].baseScore`     | The baseScore of the score in which this product_status is referenced by id.                                                                                                                                                                                    |
+| `vulnerabilities[].product_status.known_not_affected[].flags`         | Comma seperated list of corresponding flags                                                                                                                                                                                                                     |
```

Comment:
For consistency, I guess that should be an array.

Reply:
Yes I agree. I'm going to change it